content/assets/site.css

```diff
@@ -9,16 +9,22 @@ body {
   text-align: center;
   padding: 190px 0 60px 0;
   text-shadow: rgba(0, 0, 0, 0.2) 0px 2px 0px;
-}
 
-.berlin-banner {
-  background-image: url("/assets/berlin-banner.jpg");
   background-repeat: no-repeat;
```

Comment:
Including these in the default class will result in the fallback (conference without banner) not working anymore. I guess that's fine for now though.